config/marshaler.go

```diff
@@ -2,27 +2,12 @@ package config
 
```

Comment:
Please delete this file and use `json.Marshal` and `json.Unmarshal` directly.

Reply:
deleted.